### ISSUE TITLE
Add support for multiple images and emojis

### ISSUE THREAD

**Issue by Liampronan** [sudeepag/SAConfettiView#28]
I'm working on a fork that adds three new types: .images, .emoji, and .emojis. If there's any interest, I can open a PR -- let me know if so!

In the spirit of the this repo's demos, here's a (albeit low-quality) gif : 

![emoji_multi_image_confetti](https://cloud.githubusercontent.com/assets/4316904/19012041/133d486c-875e-11e6-90c2-9ca8f57715bb.gif)

**Comment by saldous** [comment]
yes please!

**Comment by jaymutzafi** [comment]
@Liampronan Did you ever make the emoji version?

**Comment by Liampronan** [comment]
I did but haven't submitted a PR -- is this library still maintained?

**Comment by jaymutzafi** [comment]
I don't know. But if you have a version I would love a peak at it. Actually all I'm trying to do is use a custom image but without having the color overlay on it and it also happens that the image I am using was taken from the bacon emoji so i wanted to see if your version helps me here.

**Comment by Liampronan** [comment]
@jaymutzafi basic implementation:
```

import SAConfettiView
import UIKit

class ViewController: UIViewController {

    override func viewDidLoad() {
        super.viewDidLoad()
        let confettiView = SAConfettiView(frame: self.view.bounds)
        confettiView.type = .Image("🥓".image())
        confettiView.colors = [UIColor.white]
        confettiView.startConfetti()
        view.addSubview(confettiView)
    }
}


extension String {
    func image() -> UIImage {
        let size = CGSize(width: 30, height: 35)
        UIGraphicsBeginImageContextWithOptions(size, false, 0);
        UIColor.clear.set()
        let rect = CGRect(origin: CGPoint.zero, size: size)
        UIRectFill(CGRect(origin: CGPoint.zero, size: size))
        (self as NSString).draw(in: rect, withAttributes: [NSFontAttributeName: UIFont.systemFont(ofSize: 30)])
        let image = UIGraphicsGetImageFromCurrentImageContext()
        UIGraphicsEndImageContext()
        return image!
    }
    
}
```

![image](https://user-images.githubusercontent.com/4316904/28907898-6eeea4ea-77d5-11e7-9988-bdc8cc36407c.png)

**Comment by jaymutzafi** [comment]
Splendid! Thank you.

**Comment by Liampronan** [comment]
👍 for images in general, you may need to ensure you call `confettiView.colors = [UIColor.white]` before `startConfetti()` -- it looks like setting colors doesn't reset the confettiView colors if it's already animating. haven't tested that out recently w/ non-emoji images but I'd imagine similar funcitonality

**Comment by jaymutzafi** [comment]
Yeah .white solved my color problem, so I'm all good! But out of curiosity I've tried "🥓".image() and it says string has no member .image()

**Comment by jaymutzafi** [comment]
Ah silly me, I messed it up. Looks good. (that's pretty clever btw, nicely done)
